docs/index.md

```diff
@@ -1,54 +1,17 @@
 
-Is the objective function maximizing DPR? [There](http://www.enworld.org/forum/showthread.php?362272-Highest-DPR-Build-Yet) are [plenty](http://www.giantitp.com/forums/showthread.php?522093-5e-Nova-or-DPR-optimization) of [threads](http://www.giantitp.com/forums/showthread.php?522096-Nova-or-DPR-optimization) that [attempt](http://www.giantitp.com/forums/showthread.php?474803-Good-sustained-dpr-builds) to [maximize](https://docs.google.com/spreadsheets/d/1Bwv7pfPC90BqZVPMu19T075-6cwkphdPJQe1d8gTVbQ/edit) dps.
-There are more [guides](http://rpgbot.net/dnd5/characters/classes/) that offer optimization advice than I can count.
+### What are we trying to optimize for?
 
-Yet, what are we *truly* trying to optimize for?
+You should maximize the probability that your party lives through an adventuring day. Page 84 of the Dungeon Master's Guide says most parties can handle 6-8 medium or hard encounters per day. The DMG also says the party will likely need to take two short rests during each of these days. At the end of the day, all of the decisions a party makes contribute to survival. The objective function needs to account for all of them (or as many as possible).
 
-Here, I propose the an optimization function: maximize the probability that your party lives through an adventuring day as defined by page 84 of the DMG.
-One such adventuring day is 6 medium difficulty encounters for a party of 4 PCs at level 5.
-The DMG suggests a short rest 1/3 and 2/3 the way through an adventuring day.
+Damage, control and defense are all measurable individually with the rules, but there isn't a way to compare how each one stacks up at the cost of the others. If you have great defenses and healing, then your enemies have a harder time killing the party. If you do great damage, then you get through your fights faster, and your enemies have a harder time killing the party. If you have great crowd control, then you can use conditions to delay or prevent enemy attacks, and your enemies have a harder time killing the party. The problem is not knowing which  gives your enemies the hardest time killing the party relative to the others. The goal of the objective function for making decisions should track the remaining resources at the end of an encounter.
```

Comment:
Correct. It should specify that there isn't a way to measure them like we do in the core rulebooks.